src/software/BUILD

```
@@ -19,6 +19,43 @@ cc_binary(
     ],
 )
 
+cc_binary(
+    name = "standalone_simulator_main",
+    srcs = ["standalone_simulator_main.cpp"],
+    deps = [
+        ":standalone_simulator",
```

Since the  standalone_simulator_main cc_binary has been moved to a different folder, it doesn't know about the standalone simulator, so we need to give the full path like this:
```suggestion
        "//simulation:standalone_simulator",
```
Similarly for the rest of the `cc_binary`'s.

You may run into [visibility issues ](https://docs.bazel.build/versions/master/visibility.html), so you may have to add `//software:__pkg__` to the visibility list.